Formula/r/redis.rb

```diff
@@ -9,8 +9,7 @@ class Redis < Formula
   head "https://github.com/redis/redis.git", branch: "unstable"
 
   livecheck do
-    url "https://download.redis.io/releases/"
-    regex(/href=.*?redis[._-]v?(\d+(?:\.\d+)+)\.t/i)
+    skip "7.2.4+ is no longer OSS"
```

Comment:
```suggestion
    url "https://download.redis.io/releases/"
    regex(/href=.*?redis[._-]v?(\d+(?:\.\d+)+)\.t/i)
    strategy :page_match do |page, regex|
      version_limit = Version.new("7.4")
      page.scan(regex).map do |match|
        next if Version.new(match[0]) >= version_limit

        match[0]
      end
    end
```

Per the previous comment, we could do something like this if we need to omit 7.4 versions for now.

Reply:
> after 7.4 is released.

7.2 will be maintained for a little while longer than that: https://github.com/redis/redis/security/policy. But yes we should adjust the livecheck to only return < 7.4.